lib/prima_opentelemetry_ex.ex

```diff
@@ -25,20 +25,34 @@ defmodule PrimaOpentelemetryEx do
     :ok
   end
 
+  defp is_enabled?(feature) do
+    Application.get_env(:prima_opentelemetry_ex, :exclude, [])
+    |> Enum.member?(feature)
+    |> Kernel.not()
```

Comment:
I'm unsure about the `Kernel.not`... I like piping stuff, but maybe a simple `!` would suffice?

```suggestion
    exclude = Application.get_env(:prima_opentelemetry_ex, :exclude, [])
    !Enum.member?(exclude, feature)
```

Reply:
That's because we wanted to use the pipe but we couldn't do
```elixir
Application.get_env(:prima_opentelemetry_ex, :exclude, [])
  |> !Enum.member?(feature)
```

I don't have a strong preference between the two, but if we use the assignment I'd rename the `exclude` variable to something like `excluded_libraries`